pycoast/cw_base.py

```diff
@@ -803,6 +826,19 @@ def add_overlay_from_config(self, config_file, area_def):
 
         return foreground
 
+    def add_overlay_from_config(self, config_file, area_def):
+        """Create and return a transparent image adding all the overlays contained in a configuration file.
+
+        :Parameters:
```

Comment:
Is this the standard docutils/rst way of specifying parameters? It's been so long I can't remember.

Reply:
Looks like it's working in the old functions: https://pycoast.readthedocs.io/en/latest/api/pycoast.html